make/build_test.mk

```diff
@@ -12,12 +12,13 @@ $(T)_TEST_ROOT := $($(T)_DIR)/test
 $(T)_TEST_DEPS := unity $(T)
 
 # Find all test_*.c files - these are our unit tests
-$(T)_TEST_SRC := $(wildcard $($(T)_TEST_ROOT)/test_*.c)
+$(T)_TEST_SRC := $(wildcard $($(T)_TEST_ROOT)/test_*.c) \
```

Comment:
We have a macro for this if you want to use it - `find_in` should do exactly what you have.